backend/graph/api/episodes.resolvers.go

```diff
@@ -199,31 +201,38 @@ func (r *episodeResolver) Files(ctx context.Context, obj *model.Episode, audioLa
 
 // Chapters is the resolver for the chapters field.
 func (r *episodeResolver) Chapters(ctx context.Context, obj *model.Episode) ([]*model.Chapter, error) {
-	i, err := r.Loaders.EpisodeLoader.Get(ctx, utils.AsInt(obj.ID))
-	if err != nil || !i.AssetID.Valid {
+	return r.getChapters(ctx, obj.ID)
+}
+
+// SkipToChapter is the resolver for the skipToChapter field.
```

Comment:
can't we use better method names, so that this comment is not necessary?

Reply:
I don't know enough about go. But `skipToChapter: Chapter @goField(forceResolver: true)` from episodes.graphqls makes me wonder if this comment is inaccurate.

Reply:
its automatic via codegen x)